djangoplicity/archives/contrib/search/forms.py

```diff
@@ -120,7 +120,6 @@ class AdvancedSearchForm (forms.Form):
 
         # take fields from options class, and sort them correctly
         fields = [(field_name, obj) for field_name, obj in options.AdvancedSearch.__dict__.items() if isinstance(obj, formsFields.Field)]
-        fields.sort(lambda x, y: cmp(x[1].creation_counter, y[1].creation_counter))
 
```

Comment:
Now. If we remove the "sort" the error is not displayed:

![image](https://user-images.githubusercontent.com/36459449/102915830-6d7e5380-4450-11eb-91cb-952d71e2b45a.png)

I did some searches (images) on my local and on the space site in order to find differences. However, there were no differences in searches:

![image](https://user-images.githubusercontent.com/36459449/102916328-5be97b80-4451-11eb-813b-a8f514a85913.png)

![image](https://user-images.githubusercontent.com/36459449/102916373-6e63b500-4451-11eb-87a5-a7f5f5fc1311.png)

![image](https://user-images.githubusercontent.com/36459449/102916414-86d3cf80-4451-11eb-95f1-696993413b5d.png)

![image](https://user-images.githubusercontent.com/36459449/102916517-b2ef5080-4451-11eb-8134-a2613571fc83.png)

What do you think?